util/session/sessionmanager.go

```diff
@@ -66,6 +66,11 @@ const (
 	autoRegenerateTokenDuration = time.Minute * 5
```

Comment:
Looks like this change is not related to the PR. Can you revert please?

Reply:
yep